db/memtable.cc

```diff
@@ -769,7 +770,8 @@ bool MemTable::Get(const LookupKey& key, std::string* value, Status* s,
                    MergeContext* merge_context,
                    SequenceNumber* max_covering_tombstone_seq,
                    SequenceNumber* seq, const ReadOptions& read_opts,
-                   ReadCallback* callback, bool* is_blob_index, bool do_merge) {
+                   ReadCallback* callback, bool* is_blob_index, bool do_merge,
+                   bool skip_filter_check) {
```

Comment:
Rather than add another parameter with default value to skip bloom filter, I'd suggest putting the code to call table_->Get in a separate function that can be called by both Get and MultiGet.